weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/csvUtils.ts

```
@@ -0,0 +1,274 @@
+  // If the value is already the correct type, return it directly
+  if (
+    (type === 'boolean' && typeof value === 'boolean') ||
+    (type === 'number' && typeof value === 'number') ||
+    (type === 'string' && typeof value === 'string')
+  ) {
+    return value;
+  }
+
+  switch (type) {
+    case 'number':
+      const num = Number(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)